From c3ece1b24c1dcf7f4513157997cedb9c6f3e92f8 Mon Sep 17 00:00:00 2001
From: Chun-wei Fan <fanchunwei@src.gnome.org>
Date: Fri, 18 Jan 2019 17:01:47 +0800
Subject: [PATCH 15/15] Update README.win32 on features and Visual Studio
 support

Let people know that up to Visual Studio 2017 is supported, and on
enabling additional features which are not enabled by default.
---
 README.win32 | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/README.win32 b/README.win32
index 08779db477..4c2464e3f0 100644
--- a/README.win32
+++ b/README.win32
@@ -156,11 +156,14 @@ Use the Microsoft compiler, cl and Make, nmake. Say nmake -f
 makefile.msc in gdk and gtk. Be prepared to manually edit various
 makefile.msc files, and the makefile snippets in build/win32.
 
-There are also VS 2008/2010 solution and project files to build GTK+, which
+There are also VS 2008~2017 solution and project files to build GTK+, which
 are maintained by Chun-wei Fan.  They should build GTK+ out of the box,
 provided that the afore-mentioned dependencies are installed.  They will
 build GDK with the Win32 backend, GTK+ itself (with GAIL/a11y built in),
-the GAIL-Util library and the gtk-demo program.
+the GAIL-Util library and the gtk-demo program.  Please also refer to the
+README file(s) that reside in build/win32 on how to enable additional features
+that are not enabled by default, such as EGL support via libANGLE, which
+emulate the GL/EGL calls using Direct3D 9/11.
 
 Please refer to the following GNOME Live! page for a more detailed ouline
 on the process of building the GTK+ stack and its dependencies with Visual
-- 
2.17.0

